internal/converter/testdata/json_fields.go

```diff
@@ -0,0 +1,25 @@
+package testdata
+
+const JSONFields = `{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "properties": {
+        "name": {
+            "type": "string"
+        },
+        "time_stamp": {
```

Comment:
https://developers.google.com/protocol-buffers/docs/proto3#json

Message field names are mapped to lowerCamelCase and become JSON object keys.

Should be `timeStamp` (might want to pick a better multi-word example name as `timestamp` is more common)

Reply:
I've updated now, let me know if it covers your use case.